### DISCUSSION TITLE
display: none %E4%B8%8E loading="lazy"

### DISCUSSION THREAD

**Discussion by giscus** [ouuan/iles-blog#135] [Comments]
# display: none %E4%B8%8E loading="lazy"

同时使用 display: none 和 loading="lazy" 可能会让内容永远加载不出来。

https://ouuan.moe/post/2022/07/display-none-and-lazy-load

**Comment by nick-cjyx9** [comment]
草，今天刚好遇见这问题，还好看过大佬博客有点印象 :smiley:
